.github/workflows/java-ci.yaml

```diff
@@ -0,0 +1,19 @@
+name: Java CI
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
```

Comment:
Nie mamy teraz testów, ale myślę że zamiast jednego kroku z fazą _package_ możemy mieć kilka z różnymi fazami, np. _test_, _verify_ itp.  
Odsyłam do [etapów mavena](https://maven.apache.org/guides/introduction/introduction-to-the-lifecycle.html).

Reply:
Jak tak teraz myślę, to najlepiej będzie zapewne rozbić to na kilka pipeline'ów. Tylko pytanie - czy będę musiał się w takim wypadku bawić w artefakty?

Reply:
Dodałem nowe kroki w yamlu. Wydają się działać